can-dom-mutate.js

```diff
@@ -29,7 +29,11 @@ function getRelatedData(node, key) {
 }
 
 function setRelatedData(node, key, targetListenersMap) {
-	var data = dataStore.get(node) || dataStore.set(node, {}).get(node);
+	var data = dataStore.get(node)
+	if (!data) {
+		dataStore.set(node, {})
+		data = {};
```

Comment:
shouldn't `data` be the thing you pass into `dataStore.set`?

Reply:
I don't think so, it's creating the entry in the store only if it doesn't exist for the key of `node`.

Reply:
Ah, but if we want it to be an object reference... I see what you're saying. Will fix.